jiant/trainer.py

```diff
@@ -322,7 +322,12 @@ def _setup_training(
                 os.mkdir(os.path.join(self._serialization_dir, task.name))
 
             # Adding task-specific smart iterator to speed up training
```

Comment:
It's okay to leave this be, but do you know what 'smart' is supposed to mean here?

Reply:
I suspect it refers to some contrast that's no longer relevant...

Reply:
[Looks like](https://github.com/nyu-mll/jiant/pull/87/files) it was added to describe the addition of a BucketIterator. Dropping w/ a44d4d5.